src/test/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println(getName() + " started her work.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println(getName() + " finished my work.");
```

Comment:
Similarly, the message should clearly indicate that a `Bulldozer` has finished working. Use 'Bulldozer' instead of 'my' to make the message more informative.